src/core/utils/request/onUnhandledRequest.ts

```diff
@@ -24,7 +24,9 @@ export async function onUnhandledRequest(
   const url = new URL(request.url)
   const publicUrl = toPublicUrl(url) + url.search
 
-  const unhandledRequestMessage = `intercepted a request without a matching request handler:\n\n  \u2022 ${request.method} ${publicUrl}\n\nIf you still wish to intercept this unhandled request, please create a request handler for it.\nRead more: https://mswjs.io/docs/getting-started/mocks`
+  const requestBody = await request.text()
```

Comment:
I also didn't know if I'd have to clone this request or not.

i.e. is the text/body read somewhere else already? Would we need to do `const requestBody = await request.clone().text()`?

Reply:
Okay, I was wrong on this one. We have some tests that show we are aiming to minimize the number of times a request is cloned due to performance reasons. Since this is (a) an internal representation of an intercepted request (i.e. _not_ the actual request your app made), and (b) you cannot affect it past this point, I think it's safe to just read its body. 

The only use case I wonder is the `request:unhandled` event, and whether I'd be able to read the request body there.